pyboto/Boto.py

```diff
@@ -1,7 +1,11 @@
 import boto3
+import io
```

Comment:
This looks good. Let's update the README, as well to match the new calls.

Reply:
Updated README.